### ISSUE TITLE
[request] mesa/21.1.5

### ISSUE THREAD

**Issue by AndreyMlashkin** [conan-io/conan-center-index#6506]
### Package Details
  * Package Name/Version: **mesa/21.1.5**
  * Website: **https://github.com/mesa3d/mesa**
  * Source code: **https://github.com/mesa3d/mesa**

### Description Of The Library / Tool
3D Graphics Library

**Comment by Croydon** [comment]
We have one in Bincrafters which can't be directly migrated beause of missing dependencies and because it did not got much love in a while (i.e. is pretty outdated in several ways)

https://github.com/bincrafters/conan-mesa

**Comment by AndreyMlashkin** [comment]
Still it would be good to port old bincrafters recipes.

**Comment by Croydon** [comment]
Yes, we are doing what we can. Contributions are always welcome 🙂

**Comment by arnoldad** [comment]
bumping this a few years later and now that there are many more packages in Conan center. Is it now more feasible to create a Conan package for mesa? (assuming that it differs from the mesa-glu package - which is not clear to me as an outsider)
